tests/ops/schemas/language/test_language.py

```diff
@@ -46,8 +47,8 @@ class SamplePydanticSchema(BaseModel):
         test_schema_instance = SamplePydanticSchema(test_prop="foo", language="pt-BR")
         assert test_schema_instance.language == SupportedLanguage.portuguese_brazil
 
-        # test that specifying an invalid language (e.g. es-MX) throws a validation error
+        # test that specifying an invalid language (e.g. es-ES) throws a validation error
         with pytest.raises(ValidationError):
             test_schema_instance = SamplePydanticSchema(
-                test_prop="foo", language="es-MX"
+                test_prop="foo", language="es-ES"
```

Comment:
i wonder how soon we'll end up adding `es-ES` 🙃

Reply:
well I think "es" was shipped to us as "es-ES"! hopefully we don't have to add it